### DISCUSSION TITLE
Camera Global Position

### DISCUSSION THREAD

**Discussion by zetyquickly** [open-mmlab/mmhuman3d#218] [Q&A] [answered]
Hello,

Could you please point out, is there a way to get camera's coordinates in the global axis or relative to the SMPL model that are being rendered? 

Is it possible to obtain during estimate_smpl.py flow?

**Comment by caizhongang** [comment, marked as answer]
Hi @zetyquickly , we hold the assumption that the camera is fixed and at the world coordinate center (i.e., the camera coordinate system is the world coordinate system). We then estimate SMPL global orientation and transl relative to camera coordinate system. Hence, if you would like to obtain the relative position of camera w.r.t. SMPL, you may inverse the transformation and obtain the translation in the new transformation matrix. 

Hope this helps!
